Simplenote/Classes/UIViewController+Simplenote.swift

```diff
@@ -26,4 +26,21 @@ extension UIViewController {
 
         leafViewController.present(self, animated: true, completion: nil)
     }
+
+    /// Attaches a children ViewController (if needed) below the specified sibling view
+    ///
+    func attachWithAnimation(to parent: UIViewController, below siblingView: UIView) {
+        parent.view.insertSubview(view, belowSubview: siblingView)
+        parent.addChild(self)
+        view.fadeIn()
+    }
+
+    /// Detaches the receiver from its parent
+    ///
+    func detachWithAnimation() {
+        view.fadeOut { _ in
+            self.view.removeFromSuperview()
+            self.removeFromParent()
+        }
+    }
```

Comment:
When adding a view controller as a child we need to call didMove after the view is added. Similar in removing.
I always forget the sequence and often consult with: 

https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html

:-)